agent/src/softwarecontaineragentadaptor.h

```diff
@@ -0,0 +1,56 @@
+#ifndef SOFTWARECONTAINERAGENTADAPTOR_H
+#define SOFTWARECONTAINERAGENTADAPTOR_H
+
+#include "softwarecontaineragent.h"
+
+namespace softwarecontainer {
+
+class SoftwareContainerAgentAdaptor :
+    public com::pelagicore::SoftwareContainerAgent_adaptor
```

Comment:
That namespace is dictated by the generated D-Bus code I think, so we'd need to change the API (the xml etc.) in order to change this.